src/main/java/com/alex/eshop/repository/CategoryRepository.java

```diff
@@ -1,10 +1,13 @@
 package com.alex.eshop.repository;
 
 import com.alex.eshop.entity.Category;
-import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.repository.PagingAndSortingRepository;
 import org.springframework.stereotype.Repository;
 
 
 @Repository
-public interface CategoryRepository extends JpaRepository<Category, Long> {
```

Comment:
**JpaRepository** extends **PagingAndSortingRepository** so, this change is reduces the capabilities of the repository